default.nix

```diff
@@ -167,29 +167,37 @@ in rec {
 
   inherit mkAssets;
 
-  serverExe = backend: frontend: assets: optimizationLevel: version:
+  serverExe = backend: frontend: assets: optimizationLevel: bundledConfig: version:
```

Comment:
might want to switch this to a struct so that we can provide defaults for things like bundledConfig

Reply:
Agreed.

Reply:
In this PR?

Reply:
Sure.